util/parseSources.py

```diff
@@ -40,5 +42,5 @@
             # We got a multiliner continuation
             pkg[cur_param] += line.lstrip()
         else:
-            print "IMMPOSSIBIRUUUU!!!!"
+            print("IMMPOSSIBIRUUUU!!!!")
```

Comment:
Is it really necessary?

Reply:
Left only system exit, without additional info output.